mappings/net/minecraft/client/gui/screen/ingame/DoubleChestScreen.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_264 net/minecraft/client/gui/screen/ingame/DoubleChestScreen
+	FIELD field_1063 playerInventory Lnet/minecraft/class_134;
+	FIELD field_1064 inventory Lnet/minecraft/class_134;
+	FIELD field_1065 rows I
+	METHOD <init> (Lnet/minecraft/class_134;Lnet/minecraft/class_134;)V
+		ARG 1 playerInventory
```

Comment:
Just `inventory` according to yarn

Reply:
there already is an argument named inventory

Reply:
I think `playerInventory` here makes enough sense, and doesn't deviate too much from yarn, considering code differences.